paddle/fluid/operators/interpolate_op.h

```diff
@@ -48,20 +51,31 @@ template <typename T>
 static void BilinearInterpolation(const Tensor& input, Tensor* output,
                                   const float ratio_h, const float ratio_w,
                                   const int in_h, const int in_w, const int n,
-                                  const int c, const int out_h,
-                                  const int out_w) {
+                                  const int c, const int out_h, const int out_w,
+                                  const bool align_corners,
+                                  const bool align_mode) {
   auto input_t = EigenTensor<T, 4>::From(input);
   auto output_t = EigenTensor<T, 4>::From(*output);
   for (int k = 0; k < out_h; k++) {  // loop for images
-    int y_n = static_cast<int>(ratio_h * k);
+    int y_n = (align_mode == 0 && !align_corners)
```

Comment:
Before line 59 (for loop):

```c++
bool align_flag = (align_mode == 0 && !align_corners);
```

Then  use `align_flag` in following codes:

```
int y_n = align_flag ? xx : xx;
```

Reply:
done